store/src/java/com/zimbra/cs/imap/ImapDaemon.java

```diff
@@ -148,6 +152,26 @@ private static boolean isZimbraImapEnabled() throws ServiceException {
         return false;
     }
 
+    private static void maybeInitEphemeralBackendExtension() throws ServiceException {
```

Comment:
I wonder if we could do with a shared method in ExtensionUtil.  Maybe some crossover with AttributeMigration.initEphemeralBackendExtension?